sources/index.yml

```diff
@@ -10,12 +10,12 @@ orbs:
 executors:
   node:
     docker:
-      - image: circleci/node:12.22-buster
+      - image: cimg/node:12.22
         user: circleci
     resource_class: medium+
   buildpack-deps:
     docker:
-      - image: circleci/buildpack-deps:buster
+      - image: cimg/base:current-22.04
```

Comment:
Switches from base debian (former LTS recently out of support) to the latest Ubuntu LTS supported by `cimg/base` (the suggested alternative from `buildpack-deps`). I can't see any reason this would be an issue though.